### PR TITLE
Enable Issue Reporting

### PR DESCRIPTION
**Pull request by mdomasevicius**
image

### CODE REVIEW

File: src/constants/urls.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) 2020 BlockDev AG
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+export const NAT_FIX_URL =
+    'http://docs.mysterium.network/en/latest/user-guide/installation/#router-configuration-for-nodes-behind-nat';
+
+export const MMN_USER_PROFILE_URL = 'https://betanet.mysterium.network/user/profile';
```
Comment (Waldz) [inline src/constants/urls.ts:10]: Can we take this from config["mmn.web-address"]  + "/user/profile"?

**Review by ignasbernotas — APPROVED** [review]

File: src/constants/urls.ts (hunk 1 above)
Comment (mdomasevicius) [reply src/constants/urls.ts:10]: I can't seem to find base URL for MMN in this config, they all point to `/api/v1` but I'd like to have single source of truth to be honest :)

image

Comment (Waldz) [reply src/constants/urls.ts:10]: Fixed: https://github.com/mysteriumnetwork/node/pull/2690

Comment (mdomasevicius) [reply src/constants/urls.ts:10]: I'll fix the source of this URL with next PR

**Pull request merged**